lib/ex_doc/formatter/html.ex

```diff
@@ -15,10 +15,21 @@ defmodule ExDoc.Formatter.HTML do
   def run(module_nodes, config) when is_map(config) do
     config = normalize_config(config)
     output = Path.expand(config.output)
-    File.rm_rf! output
-    :ok = File.mkdir_p output
+    build = Path.join(output, ".build")
+
+    if File.exists? build do
+      File.stream!(build)
+      |> Enum.map(fn(line) ->
```

Comment:
Yeah, I was thinking about moving this section into a new function. Thanks for reminding me of this :)